doc/specifications/rest/REST-API-V2.rst

```diff
@@ -228,6 +228,8 @@ Resource                                                          POST
 /content/objects/<ID>                                             .                   load content            update content meta data     delete content   copy content
 /content/objects/<ID>/<lang_code>                                 .                   .                       .                            delete language
                                                                                                                                            from content
+/content/objects/<ID>/permissions                                 .                   load content and        .                            .                .
```

Comment:
Maybe we should change this to `/content/permissions/objects/<ID>`. Location would then be `/content/permissions/locations/<path>`.